### PR TITLE
Add Episode 2 shownotes

### PR DESCRIPTION
**Pull request by garricn**

### CODE REVIEW

File: Shownotes/Episode2 [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+- [Swift Weekly Brief](https://swiftweekly.github.io)
+- [Jesse Squires](https://twitter.com/jesse_squires)
+- [iOS Dev Weekly](https://iosdevweekly.com)
+- [Ole Begemann's PR](https://github.com/SwiftCommunityPodcast/podcast/issues/15)
+- [Gorka Ercilla's PR](https://github.com/SwiftCommunityPodcast/podcast/issues/23)
+- [Mark Malström's PR](https://github.com/SwiftCommunityPodcast/podcast/issues/13)
+- [Julio César Fernández](https://github.com/SwiftCommunityPodcast/podcast/issues/21)
+- [Morten Bek Ditlevsen's comment](https://github.com/SwiftCommunityPodcast/podcast/issues/1#issuecomment-455175831)
+- [See Greg Heo's Brief Issues](https://swiftweekly.github.io/authors)
+- [PR #18](https://github.com/SwiftCommunityPodcast/podcast/pull/18)
```
Comment (BasThomas) [inline Shownotes/Episode2]: Maybe give these pull requests some more context, so it is easier to understand what they are referencing?

Comment (BasThomas) [reply Shownotes/Episode2]: No push access, so I created #30. Also adds the `.md` suffix :)

Comment (BasThomas) [reply Shownotes/Episode2]: git-fu? 😄

File: Shownotes/Episode2.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# 2: Scaling A Codeless Open Source Swift Community with Bas Broek
+
+On this episode, Garric Nahapetian, Chris Lattner, and Bas Broek use Bas's experience as the curator of the Swift Weekly Brief as a jumping off point to discuss the unique challenge of maintaining an open source project where the community does not necessarily commit a lot of code.
+
+## Hosted by
+
+- [Garric Nahapetian](https://twitter.com/garricn)
+- [Chris Lattner](https://twitter.com/clattner_llvm)
+- [Bas Broek](https://twitter.com/BasThomas)
```
Comment (lattner) [inline Shownotes/Episode2.md]: Bas is the star here, you can move him above me and maybe to the top?

File: Shownotes/Episode2.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+- [Jesse Squires](https://twitter.com/jesse_squires)
+- [iOS Dev Weekly](https://iosdevweekly.com)
+- [Ole Begemann's PR creating transcripts with AWS transcribe](https://github.com/SwiftCommunityPodcast/podcast/issues/15)
+- [Gorka Ercilla's PR about a podcast song](https://github.com/SwiftCommunityPodcast/podcast/issues/23)
+- [Mark Malström's PR about the future of Swift](https://github.com/SwiftCommunityPodcast/podcast/issues/13)
+- [Julio César Fernández about non-english Swift](https://github.com/SwiftCommunityPodcast/podcast/issues/21)
+- [Morten Bek Ditlevsen's comment](https://github.com/SwiftCommunityPodcast/podcast/issues/1#issuecomment-455175831)
+- [See Greg Heo's Brief Issues](https://swiftweekly.github.io/authors)
+- [PR #18 What is takes to produce an episode](https://github.com/SwiftCommunityPodcast/podcast/pull/18)
+- [PR #19 Episdoe Proposal Template](https://github.com/SwiftCommunityPodcast/podcast/pull/19)
```
Comment (lattner) [inline Shownotes/Episode2.md]: typo Episdoe

**Comment by lattner** [comment]
Looks great, thank you!!

**Pull request merged**